knowledge_repo/app/templates/base.html

```diff
@@ -94,7 +94,7 @@
 
             <!-- "Write a Post" button -->
```

Comment:
Let's update the comment here too if we're switching to `Add`

Reply:
👍